crates/sui-config/src/genesis.rs

```diff
@@ -317,6 +318,10 @@ impl UnsignedGenesis {
     pub fn sui_system_object(&self) -> SuiSystemState {
         get_sui_system_state(&self.objects()).expect("Sui System State object must always exist")
     }
+
+    pub fn authenticator_state_object(&self) -> Option<AuthenticatorStateInner> {
+        get_authenticator_state(&self.objects()).ok()
```

Comment:
nit: I would prefer `get_authenticator_state` to return Result<Option< AuthenticatorStateInner>> instead of converting any error to `Option::None`

Reply:
done!